fuel-tx/src/transaction/types/storage.rs

```diff
@@ -69,7 +58,7 @@ impl Distribution<StorageSlot> for Standard {
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> StorageSlot {
```

Comment:
Note: now `sample` doesn't generate from all possible values anymore, which might affect test coverage. Seems to be a non-issue for now.